caim_base/notifications.py

```diff
@@ -50,3 +53,46 @@ def notify_new_fosterer_profile(fosterer):
         context={"fosterer": fosterer},
         from_email="notifications@caim.org",
     )
+
+
+def notify_fosterer_of_animal_suggestion(suggested_animal: FosterApplicationAnimalSuggestion):
+    send_templated_mail(
+        template_name="application_animal_suggestion",
+        from_email="notifications@caim.org",
+        recipient_list=[suggested_animal.application.fosterer.email],
+        context={
+            "suggestion": suggested_animal,
+            "url_prefix": settings.URL_PREFIX,
+        },
+    )
+
+def notify_caim_of_animal_suggestion(suggested_animal: FosterApplicationAnimalSuggestion):
+    send_templated_mail(
+        template_name="application_animal_suggestion_internal",
+        from_email="notifications@caim.org",
+        recipient_list=["hello@caim.org"],
+        context={
+            "suggestion": suggested_animal,
+            "url_prefix": settings.URL_PREFIX,
+        },
+    )
+
+def notify_caim_foster_application_accepted(application: FosterApplication):
+    send_templated_mail(
+        template_name="application_accepted_internal",
+        from_email="notifications@caim.org",
+        recipient_list=["hello@caim.org"],
+        context={
+            "app": application,
+        }
+    )
+    
+def notify_caim_foster_application_rejected(application: FosterApplication):
+    send_templated_mail(
+        template_name="application_rejected_internal",
+        from_email="notifications@caim.org",
+        recipient_list=["hello@caim.org"],
```

Comment:
We re-use these emails a few time, mind extracting them to a constant at the top of the file?
```python
NOTIFICATIONS_SOURCE_EMAIL = "notifications@caim.org"
INTERNAL_NOTIFICATIONS_EMAIL = "hello@caim.org"
```
(or whatever the second one represents - is it supposed to be a notification for us?)

Reply:
hello@caim.org is an address to send notifications to ourselves AFAIK ya